### ISSUE TITLE
DatePicker calendar is not accessible with JAWS

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#8454]
## Environment

> Operating system

Windows

> Browser

Chrome v90

> Assistive technology used to verify

JAWS 2020

## Detailed description

> What version of the Carbon Design System are you using?

7.33 (I used our Storybook)

> What did you expect to happen?

The calendar dialog should open and should be navigable like a grid with the arrow keys. As the user moves around the calendar the dates should be read in an expected style "23, 24, 25"

> What happened instead?

date selection dialog (calendar drop down) improperly reads dates. Moving form 23 to the 25 of a month speaks as "2, 3, 2, 5, 2, 5". The dates are still selectable

## Notes
This issue **is not present** in Firefox. It seems to be specific to Chrome/Chromium browsers. I can reproduce the bug in the example on [FlatPickr's docs site](https://flatpickr.js.org/examples/)

**Comment by jpbyrne** [comment]
Hi @dakahn,

Just wondering if/where this one is on your roadmap?

I'm part of a team in Watson Health thats in the process of adopting Carbon, and this issue was flagged during our QA process.

**Comment by dakahn** [comment]
@jpbyrne since this bug seems to be upstream we're exploring dropping the dependency in our next major release

**Comment by grahamharper** [comment]
Any updates or plans to fix this or replace flatpickr?

**Comment by grahamharper** [comment]
Any updates or plans to fix this or replace flatpickr? There are several accessibility issues with flatpickr.
